src/main/resources/ai/privado/rulevalidator/schema/collections.json

```diff
@@ -0,0 +1,76 @@
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://example.com/object1659701478.json",
```

Comment:
Do these values in the schema, id represent something, if yes can we have some convention for them?

Reply:
The _$schema_ is a link to schema for this file.  This link states that we use version v7 defined by json-schema.org

_$id_ is just a standard identifier. Think of it as an ARN or HRN for a resource(Hence the `/`, the keys defined under this object derive the initial link and extend further).

Reply:
Ideally the value against _$id_ would lead to a hosted document, having same contents as this file. These are currently not hosted anywhere so kept it on example.com
These can be added to the tool's documentation, would be a nice to have feat.